lib/python/Components/SystemInfo.py

```diff
@@ -457,7 +457,7 @@ def getWakeOnLANType(fileName):
 BoxInfo.setItem("SoftCam", hasSoftcam())
 BoxInfo.setItem("StandbyPowerLed", fileExists("/proc/stb/power/standbyled"))
 BoxInfo.setItem("STi", SOC_FAMILY.startswith("sti"))
-BoxInfo.setItem("SuspendPowerLed", fileExists("/proc/stb/power/suspendled"))
+BoxInfo.setItem("SuspendPowerLed", fileExists("/proc/stb/power/suspendled") and MACHINEBUILD not in ("maxytecmulti", "maxytecmultise", "anadolmulti", "anadolmultitwin", "anadolmultiboxse", "axmulticombo", "axmultitwin", "axmultiboxse", "novaler4k", "novaler4kse", "novaler4kpro"))
```

Comment:
Where do you get this list of models?

Reply:
https://github.com/oe-alliance/build-enviroment/blob/5.3/Makefile

Reply:
Where do you find not SuspendPowerLed support in this makefile?

Reply:
You said its a driver bug, i think not only one of these box type have this.

Reply:
Then we need to find first all affected models and not only guessing.

Reply:
I need to remove "novaler4kpro".
All others are:
MACHINE=multibox
MACHINE=multiboxse